components/compositing/touch.rs

```diff
@@ -3,14 +3,18 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 use euclid::{Point2D, Scale, Vector2D};
-use log::warn;
+use log::{debug, warn};
 use script_traits::{EventResult, TouchId};
 use style_traits::DevicePixel;
+use webrender_api::units::{DeviceIntPoint, LayoutVector2D};
 
 use self::TouchState::*;
 
 /// Minimum number of `DeviceIndependentPixel` to begin touch scrolling.
 const TOUCH_PAN_MIN_SCREEN_PX: f32 = 20.0;
+const FLING_SCALING_FACTOR: f32 = 0.95;
+const FLING_MINIMUM: f32 = 3.0;
+const FLING_MAX: f32 = 4000.0;
```

Comment:
Do you mind adding documentation for these constants and making them consistent with `TOUCH_PAN_MIN_SCREEN_PX`?

Reply:
I added documentation and renamed `FLING_MINIMUM` to `FLING_MIN_SCREEN_PX` (not sure if that was what you had in mind). 
I also added a new todo here, since the constants are actually used with the DevicePixel unit, and would need to be multiplied with the `hidpi_factor` (but the existing touch pan constant has the same issue).